core/conn/jdbc_type2/native/SqlInterface.cpp

```diff
@@ -2570,12 +2570,9 @@ SQLRETURN ALLOCSQLMXHDLS_SPJRS(SRVR_STMT_HDL *pSrvrStmt, SQLSTMT_ID *callpStmt,
 	if (pModule->module_name == NULL)
 	{
 		DEBUG_OUT(DEBUG_LEVEL_STMT,("***pModule->module_name == NULL  Call AllocStmtForRs()"));
-/* Commenting out for now - will be looked at when SPJ is supported
- #ifdef NSK_PLATFORM
 		CLI_AllocStmtForRS(callpStmt,
```

Comment:
@DaveBirdsall Actually, CLI_AllocStmtForRS is a macro. retcode will capture the return code of "SQL_EXEC_AllocStmtForRS" (Debug.h +125). So we don't need capture the return code here.